compiler/rustc_mir/src/util/pretty.rs

```diff
@@ -1023,3 +1023,20 @@ pub fn dump_mir_def_ids(tcx: TyCtxt<'_>, single: Option<DefId>) -> Vec<DefId> {
         tcx.mir_keys(()).iter().map(|def_id| def_id.to_def_id()).collect()
     }
 }
+
+/// Calc converted u64 decimal into hex and return it's length in chars
+///
+/// ```ignore (cannot-test-private-function)
```

Comment:
Does this not just work...?

Reply:
Doctest expects public function, not private, so it can't see this.

Reply:
I would just remove the "(cannot- test-private-function)" comment then

Reply:
If i remove comment, this will emit error, see previous rust log analyzer error.

Reply:
Oh, oops. This is good then!